definition_files.rst

```diff
@@ -681,16 +680,34 @@ E.g., the ``devel`` stage in the above definition cannot copy files from the
 
 .. _apps:
 
-----
-Apps
-----
+---------
+SCIF Apps
+---------
+
+SCIF is a standard for encapsulating multiple apps into a container. A
+container with SCIF apps has multiple entry points, and you can choose
+which to run easily. Each entry point can carry out a different task
+with its own environment, metadata etc., without the need for a
+collection of different containers.
+
+Singularity implements SCIF, and you can read more about how to use it
+below.
 
-The ``%app*`` sections can exist alongside any of the primary sections (i.e.
-``%post``, ``%runscript``, ``%environment``, etc.).  As with the other sections,
+SCIF is not specfic to Singularity. You can learn more about it at the
+project's site: `<https://sci-f.github.io/>`_ which includes extended
+tutorials, the specification, and other information.
+
+SCIF %app* sections
+===================
+
+SCIF apps within a Singularity container are created using ``%app*``
+sections in a definition file. These ``%app*`` sections can exist
+alongside any of the primary sections (i.e.  ``%post``,
```

Comment:
The one difference is post - the SCIF version would be appinstall.

Reply:
Same with runscript - it’s just app run. There is a complete list in the standard doc if it’s easier to direct the user there.

Reply:
Right, those are shown in the example. The intent of this sentence...

>  "These ``%app*`` sections can exist alongside any of the primary sections (i.e.  ``%post``" i

Is that the `%app*` don't _replace_ `%post` etc. but both can be used in the same definition file. Open to rewording it.

Reply:
ohh gotcha, I just misread this!